include/SSVOpenHexagon/Core/HexagonGame.hpp

```diff
@@ -205,6 +205,7 @@ class HexagonGame
     std::optional<SwapParticleSpawnInfo> swapParticlesSpawnInfo;
     float nextPBParticleSpawn{0.f};
     float pbTextGrowth{0.f};
+    float messageTextCharacterSize{32.f};
```

Comment:
This likely needs to be reset between levels or even between resets of the same level. Maybe make it part of `HGStatus`? That should get reset automatically.